src/test/dbwrapper_tests.cpp

```diff
@@ -10,13 +10,11 @@
 #include <boost/assign/std/vector.hpp> // for 'operator+=()'
 #include <boost/assert.hpp>
 #include <boost/test/unit_test.hpp>
-                    
-using namespace std;
-using namespace boost::assign; // bring 'operator+=()' into scope
-using namespace boost::filesystem;
-         
+
+namespace io = boost::filesystem;
```

Comment:
Aliasing a namespace is quite different from `using` it, I think. I prefer removing the alias and just using the full namespace instead, in that case.